site/_data/releases.yaml

```diff
@@ -1,7 +1,12 @@
 stable:
-  name: Darga-3
-  tag: darga-3
-  filename: darga-3
+  name: darga-4.1
+  tag: darga-4.1
+  filename: darga-4.1
+
+prerelease:
+  name: euwe-1-beta1.3
+  tag: euwe-1-beta1.3
+  finename: euwe-1-beta1.3
```

Comment:
@Fryguy Currently the download page is 100% manual, so this won't actually change anything.